### ISSUE TITLE
framework_time is growing over time

### ISSUE THREAD

**Issue by benvanwerkhoven** [KernelTuner/kernel_tuner#188]
The idea behind framework_time it includes the 'overhead' time of the tuning framework per configuration. However on a longer run (several minutes) I was running just now I noticed that framework time seems to be increasing over time, so something seems to be wrong in how we account the framework time per configuration.

**Comment by fjwillemsen** [comment]
Is it possible that appending to the cache gets increasingly more expensive? Does it make any difference which strategy is being used?

**Comment by benvanwerkhoven** [comment]
I was using random, so I wouldn't expect the strategy to be of much influence. I looked at the code and indeed it's rather hard to see how things could accumulate in the time as it is being calculate. The more logical explanation is indeed as you suggest that some of the things that happen as part of framework_time do get more expensive over time. Not sure what it is yet, could be storing to the cache indeed, or something similar.

**Comment by benvanwerkhoven** [comment]
I've just tested a different method for storing to the cache, keeping the file handle open instead of opening the file again every time that we want to append a single line. Turns out this leads to the exact same framework time that is increasing over time. 

I also tested with brute force as the strategy and that does matter, which is strange because we want to record the time spent in the strategy as part of strategy_time and not framework_time.
